### ISSUE TITLE
 [Adaptive Cards - iOS - AdaptiveCard.json]: After activating the 'Comment' button, voice over focus is not landing on the 'Enter your comment Edit field' control.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8424]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'AdaptiveCard.json' and activate it.
7. Card will appear.
8. Navigate to the 'Comment' button and double tap on it and observe the issue.

**Actual Result:** 
After activating the 'Comment' button, Voice over focus is not landing on the 'Edit Box'. It is landing on the top of the screen.

**Expected Result:**
Voice over focus should be land on the 'Edit Box' when user activates the 'Comment' button.

**Observation:**
Same issue is observed with Keyboard as on activating the 'Comment' button keyboard focus is remain same on the 'Comment' button.

**Note:**
This issue is being reported throughout the application.

**User Impact:**
It will impact the screen reader user as they have to put more efforts to renavigate on the activated control.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

### Screenshots

https://user-images.githubusercontent.com/95913748/228550089-166c2171-02eb-4cab-9a8d-7dd02956f878.mov

### Card JSON

```JSON
NA
```
